### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Jimenarosello**
### **Summary**
Creation of an initial controller 
Creation of book model 

### **Trello card**
https://trello.com/c/CrgLb5Pe/22-crear-un-controller-y-una-vista-de-bienvenida

### CODE REVIEW

File: src/main/java/Wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        this.image = image;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+    }
+
+    public Book(String author, String image, String title, String subtitle, String publisher, String year,
```
Comment (Mujics) [inline src/main/java/Wolox/training/models/Book.java]: why are you creating this constructor?

Comment (Jimenarosello) [reply src/main/java/Wolox/training/models/Book.java]: I understood from the trello card that the constructor is required by JPA. Isn't it mandatory?

Comment (Mujics) [reply src/main/java/Wolox/training/models/Book.java]: the mandatory contructor is the empty one
`public Book() {}`

File: application.properties [hunk 2]
```diff
@@ -0,0 +1,10 @@
+server.port = 8081
+spring.thymeleaf.cache=false
```
Comment (Mujics) [inline application.properties]: remove the thymeleaf, application and datasource variables. 
They are not necessary :)

File: src/main/java/Wolox/training/TrainingApplication.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package Wolox.training;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.*;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@EnableJpaRepositories("org.baeldung.persistence.repo")
```
Comment (Mujics) [inline src/main/java/Wolox/training/TrainingApplication.java]: remove this as well, just leave the SpringBootApplication annotation

File: application.properties (hunk 2 above)
Comment (Mujics) [inline application.properties]: remove the thymeleaf, application and datasource variables.
They are not necessary :)

**Comment by Mujics** [comment]
Just a few deletes and it's ready to merge :D

**Review by Mujics — APPROVED** [review]

**Pull request merged**